crates/cdk/src/client.rs

```diff
@@ -74,6 +75,7 @@ impl HttpClient {
     }
 
     /// Get Active Mint Keys [NUT-01]
+    #[instrument(skip(self))]
```

Comment:
```suggestion
    #[instrument(skip(self), fields(mint_url = %mint_url))]
```

I think we should format the url as display. its a bit verbose displaying it as debug like default. 
![image](https://github.com/cashubtc/cdk/assets/8606367/46064c37-dc23-4d9f-943f-fe40c03ee055)
vs
![image](https://github.com/cashubtc/cdk/assets/8606367/9ab3441e-52ab-464b-833f-29ec72d80d78)